Example/RxRealm/ViewController.swift

```diff
@@ -32,23 +35,28 @@ class ViewController: UIViewController {
         //Observable<Array<Lap>>
         let laps = realm.objects(Lap).sorted("time", ascending: false).asObservableArray()
         
-        laps.map {array in
-            return array.prefix(3) //get array slice of the 3 most recent items
-        }
+        laps
         .bindTo(tableView.rx_itemsWithCellIdentifier("Cell", cellType: UITableViewCell.self)) {row, element, cell in
             cell.textLabel!.text = formatter.stringFromDate(NSDate(timeIntervalSinceReferenceDate: element.time))
         }.addDisposableTo(bag)
 
-        //start adding laps
-        addLap()
-    }
-    
-    func addLap() {
-        let realm = try! Realm()
-        try! realm.write {
-            realm.add(Lap())
-        }
+        
+        addOneItemButton.rx_tap
+            .map {
+                return Lap()
+            }
+            .bindTo(realm.rx_addObject())
+            .addDisposableTo(bag)
+        
+        addTwoItemsButton.rx_tap
+            .map {
+                return [
+                    Lap(),
+                    Lap()
+                ]
+            }
```

Comment:
I'm two hands up for RxSwift style guid =) 
I think we could create a repo under RxSwiftComunity so everybody could participate in discussion

And yeh, I think it could be simplify a bit, I personally like when every operator begins with a new line :)

```swift

addTwoItemsButton.rx_tap
            .map {
                return [Lap(), Lap()]
            }

```

If you think this would be nicer, I can easily change it =)